livery.gemspec

```diff
@@ -23,11 +23,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "betterlint", "~> 1.4"
-  spec.add_runtime_dependency "rubocop", "~> 1"
+  spec.add_runtime_dependency "rubocop", "~> 1.63.5"
   spec.add_runtime_dependency "rubocop-gitlab-security", "~> 0.1"
   spec.add_runtime_dependency "rubocop-rails", "~> 2"
   spec.add_runtime_dependency "rubocop-rspec", "~> 2"
-  spec.add_runtime_dependency "standard", "~> 1"
+  spec.add_runtime_dependency "standard", "~> 1.36.0"
```

Comment:
why do we have both?

Reply:
I'm honestly not sure...? I know standard is like a superset of rubocop, and so its like a "yes standard, and additional rubocops".

Reply:
I guess i thought it was an alternative...

Reply:
@jayzes this occurs to me now this is a public repository, do you recall the interplay between standardrb and rubocop?